gluoncv/data/mscoco/segmentation.py

```diff
@@ -49,7 +49,9 @@ def __init__(self, root=os.path.expanduser('~/.mxnet/datasets/coco'),
             print('train set')
             ann_file = os.path.join(root, 'annotations/instances_train2017.json')
             ids_file = os.path.join(root, 'annotations/train_ids.mx')
-            self.root = os.path.join(root, 'train2017')
+            # self.root = os.path.join(root, 'train2017')
+            self.part1_root = os.path.join(root, 'train2017_part1')
+            self.part2_root = os.path.join(root, 'train2017_part2')
```

Comment:
what is part1 and part2 used for?

Reply:
Oops, that was some of my changes for local experiments. Sorry for letting it slip into the PR. I have reverted these changes.